projects/storefrontapp/src/environments/productconfig/productconfig.feature.enabled.ts

```diff
@@ -0,0 +1,6 @@
+import { ProductConfigurationModule } from '@spartacus/productconfig';
```

Comment:
You can drop this file as the new mechanism doesn't use it.
Before you delete it, I suggest you copy its contents to `projects/storefrontapp/src/environments/productconfig/productconfig.feature.ts`